wallet-new/src/Cardano/Wallet/Demo.hs

```diff
@@ -0,0 +1,617 @@
+{-# LANGUAGE LambdaCase #-}
+
+{-|
+
+# Demo
+
+## Getting Started
+
+This module contains a few helpers to quickly start a demo cluster of wallet
+nodes. It is designed to remove all the overhead of setting up a configuration
+and an environment and to _just work_, out-of-the-box. Minor configuration
+adjustments are however possible via environment variables. A few things to
+know:
+
+- ENV variables define "base" arguments for a node.
+  When it makes sense (.e.g `DB_PATH`, `LOG_CONFIG`), these "base" arguments
```

Comment:
will you also support a prefix where logs are put?
`--logs-prefix FILEPATH   Prefix to logger output path.`

Reply:
Since it reuses the CLI parsers, everything that is supported by the CLI will be seemingly supported by this. :+1: